resources/aux/keycloak-template

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
+
+services:
```

Comment:
for the moment I'll ignore those config as they need to be rechecked in other tickets

Reply:
removed the extra configs (although i just copied those from examples out there)